src/test/java/models/testPlayer.java

```diff
@@ -121,6 +121,14 @@ public void testSplit(){
 
 
     }
+    @Test
+    public void testDoubleDown(){
+        Player p = new Player();
+        p.setBet(250);
+        p.doubleDown(300);
+        assertNotEquals(250, p.bet);
```

Comment:
I would change this test to test for the expected bet value. This way, we can see that you're getting the correct value.